api/v1/models/AttendeeWebsite.js

```diff
@@ -0,0 +1,21 @@
+const Model = require('./Model');
```

Comment:
This is what extrainfo should be used for.

Reply:
Created new ExtraInfo model in 4dbfa1b32bc2b1277c64598623b3875edbcfa8f1